dags/ethereumetl_airflow/build_parse_logs_dag.py

```diff
@@ -0,0 +1,168 @@
+from __future__ import print_function
+
+import json
+import logging
+import os
+import time
+from datetime import datetime, timedelta
+from glob import glob
+
+from airflow import models
+from airflow.contrib.operators.bigquery_operator import BigQueryOperator
+from airflow.operators.python_operator import PythonOperator
+from airflow.operators.sensors import ExternalTaskSensor
+from google.api_core.exceptions import Conflict
+from google.cloud import bigquery
+from eth_utils import event_abi_to_log_topic
+
+logging.basicConfig()
+logging.getLogger().setLevel(logging.DEBUG)
+
+dags_folder = os.environ.get('DAGS_FOLDER', '/home/airflow/gcs/dags')
+
+def abi_to_event_topic(abi):
+    return '0x' + event_abi_to_log_topic(abi).hex()
+
+def build_parse_logs_dag(
```

Comment:
Let's make this function accept a list of table definitions, then create two separate DAGs for 0x and ENS: ethereum_parse_ens_dag.py, ethereum_parse_zeroex_dag.py, which call the builder with the respective list of definitions. This way we can have a separate DAG per Dapp which should be easier to maintain.

Reply:
Done. I'm using the folders inside `table_definitions` to generate the DAGs. That way we don't have to maintain a separate list of table definitions. Let me know if you'd like me to do this in a different way.